src/bluesearch/entrypoint/database/topic_extract.py

```diff
@@ -168,6 +171,30 @@ def run(
                     },
                 }
             )
+
+    elif source in {"biorxiv", "medrxiv"}:
+        for path in inputs:
+            logger.info(f"Processing {path}")
+            topic, journal = extract_info_from_zipfile(path)
+            all_results.append(
+                {
+                    "source": journal,
+                    "path": str(path.resolve()),
+                    "topics": {
+                        "article": {
+                            "Subject Area": topic,
+                        },
```

Comment:
`extract_info_from_zipfile(path)` is extracting the Subject Area and the `journal title`. Do you know if there is an easy way to extract some topic information about the journal ? Do you think it could be something worth to investigate ?

Reply:
Great point. I did not investigate it all. Do you think it would be worth investigating? Intuitively, I would guess that article level topic should be more specific than the journal level one.

Reply:
As discussed during sprint, let's keep it that way :)